crates/jrsonnet-evaluator/src/builtin/mod.rs

```diff
@@ -854,6 +856,46 @@ fn builtin_ascii_lower(
 	})
 }
 
+fn builtin_member(context: Context, _loc: Option<&ExprLocation>, args: &ArgsDesc) -> Result<Val> {
+	parse_args!(context, "member", args, 2, [
+		0, arr: ty!((array | string));
+		1, x: ty!(any);
+	], {
+		match arr {
+			Val::Str(s) => {
+				let x = x.try_cast_str("x")?;
```

Comment:
Not sure about this argument.

Reply:
Both of those arguments checked [here](https://github.com/CertainLach/jrsonnet/blob/master/crates/jrsonnet-stdlib/src/std.jsonnet#L664)

As for try_cast_tr first argument - it is used in error message, so just write `x should be string`